sbin/functions.sh

```diff
@@ -943,9 +943,11 @@ extract_tarball()
 {
 	local tarball_src="$1"
 	local destination="$2"
+	local extra_opts=""
 
-	# tar -jxf ${tarball_src} -C ${destination} > /dev/null 2>&1 &
-	tar -jxf ${tarball_src} -C ${destination} &
+	[ $DO_IMA_SIGN -eq 1 ] && extra_opts="--xattrs --xattrs-include=security\\.ima"
```

Comment:
The content of --xattrs-include is intercepted as regx. So \\. is accurate, avoiding matching security_ima or the similar.

I will address all your comments in V2. Thanks.